scripts/docker/readme.md

```diff
@@ -12,13 +12,18 @@ The docker image for the astrobee FSW is divided throughout 2 docker files.
 
 Available docker files:
 
-    astrobee_base_xenial - Contains installation of all astrobee dependencies the Ubuntu 16.04 + ROS kinetic setup.
-    astrobee_base_bionic - Contains installation of all astrobee dependencies the Ubuntu 18.04 + ROS melodic setup.
-    astrobee_base_focal - Contains installation of all astrobee dependencies the Ubuntu 20.04 + ROS noetic setup.
+- `astrobee_base.Dockerfile` - Contains installation of all astrobee dependencies the Ubuntu + ROS setup.
+- `astrobee.Dockerfile` - Builds the astrobee FSW code on top of astrobee_base.
 
-    astrobee_xenial - Builds the astrobee FSW code on top of astrobee_base_xenial.
-    astrobee_bionic - Builds the astrobee FSW code on top of astrobee_base_bionic.
-    astrobee_focal - Builds the astrobee FSW code on top of astrobee_base_focal.
+The Docker files accept the following version args (note that they must match up):
+
+- `UBUNTU_VERSION` - The version of Ubuntu to use. Valid values are "16.04", "18.04", and "20.04". Defautls to "16.04". 
```

Comment:
It actually defaults to the current OS in the build.sh with:
os=`cat /etc/os-release | grep -oP "(?<=VERSION_CODENAME=).*"`

Reply:
Oh yeah. I was thinking of the `ARG`s inside the docker files that default to `16.04`. Should add a blurb about `build.sh`. My system is `manjaro` so I gotta specify the arg for `build.sh` in my case

Reply:
Oh, I forgot: because my OS is Manjaro Linux, then it still falls back to the default of `16.04`/`kinetic` because the conditional checks only try to match against bionic or focal. So the script will use xenial if the OS is not bionic or focal, which works out. In my case the default is xenail.

Reply:
Astrobee doesn't currently support any other operating system besides ubuntu. (https://nasa.github.io/astrobee/html/install-nonNASA.html)

Reply:
True that! That's why I'm using the Docker containers. Everything is running in Ubuntu inside the Docker containers.

I was just mentioning that because I'm not on Ubuntu when I run `build.sh`, the [conditional check in `build.sh`](https://github.com/nasa/astrobee/pull/267/files#diff-be45b1c53fa8ef7864c013a77325ed29aca3eeae2be39a37e107d63e0d6134d1L55-L68) will default to `xenial` for the docker container because of the [`os` variable creation](https://github.com/nasa/astrobee/pull/267/files#diff-be45b1c53fa8ef7864c013a77325ed29aca3eeae2be39a37e107d63e0d6134d1R31).

Reply:
Hmmm.... But maybe I should just make a generic Ubuntu container to run all the containers in. That will probably work best. I haven't tried that yet, I've just been running all these containers directly in Manjaro Linux.

Reply:
I think your setup is fine, you just need to be aware that it is not the standard and the documentation will not reflect this use case.